src/main/java/com/pszymczyk/ReservationSummary.java

```diff
@@ -0,0 +1,5 @@
+package com.pszymczyk;
```

Comment:
czy to jest placeholder ?

Reply:
To jest reklama:
tanie szkolenia - http://pszymczyk.com/